package.json

```diff
@@ -32,6 +33,8 @@
     "ol": "^6.14.1",
     "react": "18.1.0",
     "react-dom": "18.1.0",
+    "test": "jest",
```

Comment:
What is that?

Reply:
not me ! .... I swear w/

it is Simon :P

Reply:
Dan, that's what I was telling you! In my second commit I realized that I put the script at the wrong place that you don't forget to solve that conflict!